packages/webapp/src/containers/SensorReadings/saga.js

```diff
@@ -169,6 +169,14 @@ export function* getSensorsReadingsSaga({ payload }) {
     );
     const lastUpdatedReadingsTime = moment(lastUpdated).startOf('day').fromNow();
 
+    let xAxisLabel = '';
+    if (Object.keys(ambientDataWithSensorsReadings).length) {
+      const startDateObj = new Date(+Object.keys(ambientDataWithSensorsReadings)[0] * 1000);
+      const endDateObj = new Date(+Object.keys(ambientDataWithSensorsReadings).at(-1) * 1000);
```

Comment:
It would probably be best to put the `Object.keys()` result here into a variable rather than calling the `Object.keys()` function three times.

Reply:
resolved